lib/index.js

```diff
@@ -185,6 +189,9 @@ Server.prototype = {
     const httpPort = wsConfig.http ? wsConfig.http.port : undefined;
     const httpsPort = wsConfig.https ? wsConfig.https.port : undefined;
     const cookiePort = httpsPort ? httpsPort : httpPort;
+    const mediationLayer = wsConfig.mediationLayer;
+    const zoweInstance = process.env.ZOWE_INSTANCE || 'default';
+    const sessionCookieName = mediationLayer.enabled ? `connect.sid.${zoweInstance}` : `connect.sid.${cookiePort}`;
```

Comment:
I don't know if people currently customize ZOWE_INSTANCE, even though they should, so there are many that are ZOWE_INSTANCE=1. Is there a better way to share a cookie name for a cluster without it being the instance?

Reply:
For Eureka statuses I introduced constants.

Reply:
As for cookie name: maybe we can go with `connect.sid`. @1000TurquoisePogs  What do you think?

Reply:
This is ok for now.